examples/pong_tutorial_03/systems/paddle.rs

```diff
@@ -1,36 +1,30 @@
 use crate::pong::{Paddle, Side, ARENA_HEIGHT, PADDLE_HEIGHT};
 use amethyst::{
     core::transform::Transform,
-    derive::SystemDesc,
-    ecs::prelude::{Join, Read, ReadStorage, System, SystemData, WriteStorage},
-    input::{InputHandler, StringBindings},
+    ecs::{Runnable, SystemBuilder},
```

Comment:
Isn't it part of specs?

Reply:
Amethyst re-exports the specs api as amethyst::ecs. It is still specs but you don't import from specs directly.

Reply:
in the legion_v2 branch, the `ecs` module refers to legion